docs/new-architecture-app-modules-android.md

```diff
@@ -212,10 +266,47 @@ public class MyApplication extends Application implements ReactApplication {
 }
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+class MyApplication : Application(), ReactApplication {
+    private val mReactNativeHost: ReactNativeHost =
+        object : ReactNativeHost(this) {
+            /* ... */
+            val useDeveloperSupport: Boolean
+                get() {
+                    /* ... */
+                }
+
+            /* ... */
+            protected val packages: List<Any>
+                protected get() {
+                    /* ... */
+                }
+
+            /* ... */
+            protected val jSMainModuleName: String
+                protected get() {
+                    /* ... */
+                }
+            protected val reactPackageTurboModuleManagerDelegateBuilder:
+                ReactPackageTurboModuleManagerDelegate.Builder
+                protected get() = Builder()
```

Comment:
I would not use `val` here but just `fun`

Reply:
done